src/SourceBuild/content/eng/tools/BinaryToolKit/BinaryToolCli/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using BinaryTool;
+
+namespace BinaryToolCli;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        if (args.Length < 2 || args.Length > 6)
+        {
+            Console.WriteLine("Usage: BinaryToolCli <target-directory> <output-report-directory> [--allowed-binaries-keep-file <file> | --keep <file>] [--allowed-binaries-remove-file <file> | --remove <file>]");
+            return;
+        }
+
+        var targetDirectory = args[0];
+        var outputReportDirectory = args[1];
+        var allowedBinariesKeepFile = "";
+        var allowedBinariesRemoveFile = "";
+
+        for (int i = 2; i < args.Length; i++)
+        {
+            switch (args[i])
```

Comment:
Should we have a `default` case and throw when an unknown option is specified. For instance when you misspell one of the optional arguments, it's very hard to find out that it's happening.